docs/installation.md

```diff
@@ -3,7 +3,12 @@
 If you prefer video, [check out our demo (2 min)](https://www.youtube.com/watch?v=fr5V5EWVcyM&lc=UgxM_ZzFiFYfjo1ADU54AaABAg)
 
 ## Go to your repository and create an issue
-The issue title should start with "Sweep: ". For issues and PRs, call Sweep using the Sweep label or by prefixing your text with "Sweep: "
+To create a Sweep issue, follow these steps:
```

Comment:
I don't want this. Keep the original but just break "Sweep: " into a code block that the user can click to copy.